servers/src/main/java/tachyon/master/file/meta/InodeFile.java

```diff
@@ -89,6 +90,32 @@ public FileInfo generateClientFileInfo(String path) {
   }
 
   /**
+   * Reinitializes the inode file.
+   */
+  public void reinit() {
```

Comment:
I don't think that reinit vs. reset explains what the function does or does not do. I think the method JavaDoc should explain that. As for the method name, I'd prefer ``reset``.